va/meson.build

```diff
@@ -101,12 +101,12 @@ if WITH_DRM
   deps = [ libdrm_dep ]
 
   libva_drm_args = []
-  if get_option('with_x11') != 'no' and x11_dep.found()
+  if WITH_X11
     libva_drm_sources += [ 'drm/va_drm_auth_x11.c' ]
     libva_drm_args += [
       '-DLIBVA_MAJOR_VERSION=@0@'.format(libva_major_version)
     ]
-    deps += [ x11_dep ]
+    deps += [ x11_deps ]
```

Comment:
I'm not sure if makes sens to link with all x11-related dependencies for drm

Reply:
By default meson will pass `-Wl,--as-needed`, thus the linker will discard any unneeded DSOs.
Plus it makes little sense to have a separate dependency, just for this.

Reply:
I'm actually more tempted at removing the xlib dependency in the first place.

Reply:
Dependency removed. This hunk is no longer relevant.